src/FakeRest.js

```diff
@@ -1,11 +1,7 @@
-import Server from 'Server';
-import FetchServer from 'FetchServer';
-import Collection from 'Collection';
-import Single from 'Single';
+import { Server } from 'Server';
+import { FetchServer } from 'FetchServer';
+import { Collection } from 'Collection';
+import { Single } from 'Single';
 
-export default {
-    Server: Server,
-    FetchServer: FetchServer,
-    Collection: Collection,
-    Single: Single
-}
+export { Server, FetchServer, Collection, Single };
+export default { Server, FetchServer, Collection, Single };
```

Comment:
I don't understand why you need both export and export default

Reply:
To not break backward compatibility by forcing users to import like this:
```
import * as FakeRest from 'fakerest';
```

This way, they can keep their old import:
```
import FakRest from 'fakerest';
```